module/os/linux/spl/spl-kmem-cache.c

```diff
@@ -91,7 +91,7 @@ MODULE_PARM_DESC(spl_kmem_cache_max_size, "Maximum size of slab in MB");
  * of 16K was determined to be optimal for architectures using 4K pages and
  * to also work well on architecutres using larger 64K page sizes.
  */
-static unsigned int spl_kmem_cache_slab_limit = 16384;
+static unsigned int spl_kmem_cache_slab_limit = SPL_MAX_KMEM_ORDER_NR_PAGES * PAGE_SIZE;
```

Comment:
```suggestion
static unsigned int spl_kmem_cache_slab_limit =
    SPL_MAX_KMEM_ORDER_NR_PAGES * PAGE_SIZE;
```

80 character limit.  It looks like the style checker missed this because it happens to be in a `/* BEGIN CSTYLED */` block.